### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by bcaglayan**
Add README

### CODE REVIEW

**Review by serkan-ozal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
+      - run: npm install
+      
+      - name: Thundra Foresight JS Instrumentation
+        uses: thundra-io/thundra-foresight-js-action@v1
+        with:
+          apikey: ${{ secrets.THUNDRA_APIKEY }}
+          project_id: ${{ secrets.THUNDRA_PROJECT_ID }}
+          
+          # Modify or remove the command
+          # Default is "npm test"
```
Comment (rwxdash) [inline README.md:43]: We removed the default value to handle separate runs. This might confuse people.

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+## Parameters
+
+| Name                      | Requirement       | Default                  | Description
+| ---                       | ---               | ---                      | ---
+| apikey                    | Required          | ---                      | Thundra API Key
+| project_id                | Required          | ---                      | Your project id from Thundra. Will be used to filter and classify your testruns.
+| framework                 | Optional          | jest                     | Test framework type. Currently only jest framework is allowed.
+| environment               | Optional          | ---                      | You can specify Jest test environment using with jsdom or node values. For jest which is >= 27.0.0, default test environment is node. For this reason, Thundra will set its test environment to jest-environment-node. For jest which is < 27.0.0, default test environment is jsdom.  For this reason, Thundra will set its test environment to jest-environment-jsdom.
+| command                   | Required          | ---                      | The Npm / Yarn command you want to run.
```
Comment (rwxdash) [inline README.md:99]: The command is now optional.

File: README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+| environment               | Optional          | ---                      | You can specify Jest test environment using with jsdom or node values. For jest which is >= 27.0.0, default test environment is node. For this reason, Thundra will set its test environment to jest-environment-node. For jest which is < 27.0.0, default test environment is jsdom.  For this reason, Thundra will set its test environment to jest-environment-jsdom.
+| command                   | Required          | ---                      | The Npm / Yarn command you want to run. 
+| append_thundra_arguments  | Required          | true                     | If value is true, Thundra will try to append required argument to your command. If value is false, environment variable `THUNDRA_JEST_ARGUMENTS` must be passed to script as argument manually.
+| agent_version             | Required          | ---                      | A specific version Thundra Node Agent you want to use should be defined here. If this value is not specified latest version of Thundra Node Agent will be use.
```
Comment (rwxdash) [inline README.md:101]: Is this required as well? agent_version should not be a requirement.

File: README.md [hunk 4]
```diff
@@ -1 +1,101 @@
...
+| project_id                | Required          | ---                      | Your project id from Thundra. Will be used to filter and classify your testruns.
+| framework                 | Optional          | jest                     | Test framework type. Currently only jest framework is allowed.
+| environment               | Optional          | ---                      | You can specify Jest test environment using with jsdom or node values. For jest which is >= 27.0.0, default test environment is node. For this reason, Thundra will set its test environment to jest-environment-node. For jest which is < 27.0.0, default test environment is jsdom.  For this reason, Thundra will set its test environment to jest-environment-jsdom.
+| command                   | Required          | ---                      | The Npm / Yarn command you want to run. 
+| append_thundra_arguments  | Required          | true                     | If value is true, Thundra will try to append required argument to your command. If value is false, environment variable `THUNDRA_JEST_ARGUMENTS` must be passed to script as argument manually.
```
Comment (rwxdash) [inline README.md:100]: Maybe this should be marked optional as well.

File: README.md [hunk 5]
```diff
@@ -1 +1,100 @@
...
+
+## Parameters
+
+| Name                      | Requirement       | Default                  | Description
+| ---                       | ---               | ---                      | ---
+| apikey                    | Required          | ---                      | Thundra API Key
+| project_id                | Required          | ---                      | Your project id from Thundra. Will be used to filter and classify your testruns.
+| framework                 | Optional          | jest                     | Test framework type. Currently only jest framework is allowed.
+| environment               | Optional          | ---                      | You can specify Jest test environment using with jsdom or node values. For jest which is >= 27.0.0, default test environment is node. For this reason, Thundra will set its test environment to jest-environment-node. For jest which is < 27.0.0, default test environment is jsdom.  For this reason, Thundra will set its test environment to jest-environment-jsdom.
+| command                   | Optional          | ---                      | The Npm / Yarn command you want to run.
```
Comment (serkan-ozal) [inline README.md:98]: Is command optional or required? As far as I see, it doesn't have a default value so shouldn't it be required?

**Review by rwxdash — APPROVED** [review]

**Review by serkan-ozal — APPROVED** [review]

**Pull request merged**